src/tiledb/cloud/vcf/split.py

```diff
@@ -0,0 +1,343 @@
+"""Split samples from multi-sample VCF."""
+
+from typing import Mapping, Optional, Sequence
+
+from tiledb.cloud.dag import DAG
+
+
+def ls_samples(
+    vcf_uri: str,
+    config: Mapping[str, str],
+) -> list[str]:
+    """List samples in an aggregate VCF.
+
+    Args:
+        vcf_uri:
+            S3 path to aggregate VCF.
+        config:
+            TileDB config params.
+
+    Returns:
+        Samples included in VCF.
+    """
```

Comment:
Does this comment style work with the doc generator? I know we use sphinx style in other places.

Reply:
Sphinx picks it up nicely, but you're right I should assimilate with other doc style and was just a force of habit

Reply:
Quarto did not like that formatting, thank you for noticing this, should be fixed now